src/MICore/Transports/LocalTransport.cs

```diff
@@ -29,14 +29,14 @@ public override void InitStreams(LaunchOptions options, out StreamReader reader,
             proc.StartInfo.WorkingDirectory = miDebuggerDir;
 
             //GDB locally requires that the directory be on the PATH, being the working directory isn't good enough
-            if (proc.StartInfo.Environment.ContainsKey("PATH"))
-            {
-                proc.StartInfo.Environment["PATH"] = proc.StartInfo.Environment["PATH"] + ";" + miDebuggerDir;
-            }
+            ProcessStartInfo processStartInfo = proc.StartInfo;
```

Comment:
BTW: I fixed a bug here that you will have a merge conflict with. See Commit 7cb30a6ecdb6fe8ad3ada63052684e6f3f513108.